app/components/subject-viewer.cjsx

```diff
@@ -179,7 +179,7 @@ module.exports = React.createClass
                 <i className="fa fa-info-circle fa-fw"></i>
               </button>{' '}
             </span>}
-          {if @props.project? and @props.subject?
+          {if @props.subject?
```

Comment:
This needs to be tested a bit more. The check for the project is necessary for the FavoritesButton. The FavoritesButton will only work if there's a project; the favorites collection is for a project. It won't work outside of project context. 

Before, the project set to the parent state was the one being passed down. It was determined in either two ways: the project linked to the project or the project the subject belonged to. I think we'll have to be careful for the FavoritesButton that it gets the correct project, that is the project  that the favorites collection belongs to for that user. I think perhaps the code as it was was written to get the FavoritesButton to work correctly, but was causing buggy behavior for the Collections Manager/Creating other collections. Perhaps we should separate out the props and naming for project context vs the project the favorites collection belongs to and make sure the FavoritesButton is only visible when in a project context.